content/events/devcon24/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "[Upcoming Event] Espressif DevCon24"
+date: 2024-06-11
+draft: false
+description: "Espressif Systems Developer Conference 24"
+tags: ["event", "announcement", "devcon"]
+showDate : false
+---
+
+{{< badge >}}
+New event!
+{{< /badge >}}
+
+## DevCon24
+
+Espressif DevCon24, the third edition of Developers conference made by Developers for Developers!
+
+{{< alert icon="eye">}}
+**The DevCon24, will take place online from September 3rd to 5th 13.00 to 18.00 CEST.**
+{{< /alert >}}
+
+During the three-day online conference, we will provide you with nearly 30 talks created by Espressif technical experts and its partners, covering a wide range of topics, including Thread, Matter, ESP-IDF, Arduino Core for ESP32, ESP-Mesh-Lite + ESP RainMaker Cloud solution, RUST and more.
```

Comment:
Not clear what `its partners` refers to. Maybe company partners or community members?